vars/daosPackagesVersion.groovy

```diff
@@ -0,0 +1,96 @@
+// vars/daosPackagesVersion.groovy
+
+/**
+ * daosPackagesVersion.groovy
+ *
+ * daosPackagesVersion variable
+ */
+
+/**
+ * Method to return the version of the DAOS packages
+ */
+
+import groovy.transform.Field
+
+@Field static rpm_version_cache = ""
+
+String daos_latest_version(String next_version, String repo_type='stable') {
+    String v = sh(label: "Get RPM packages version",
+                  script: 'repoquery --repofrompath=daos,' + env.REPOSITORY_URL +
+                          env["DAOS-STACK-EL-7-${repo_type}-REPO"].toUpperCase() +
```

Comment:
You are uppercasing the returned URL, it is the environment variable name that needs to be in upper case.

Reply:
I don't think so.  The `toUpperCase()` should only being applied to `env["DAOS-STACK-EL-7-${repo_type}-REPO"]`:

```groovy
String foo="foo"
String bar="bar"
println(foo + bar.toUpperCase())
```
results in:

```
fooBAR
```

As you can see, the `toUpperCase()` method is applied to the most direct string, not the composed string.  IOW, the `toUpperCase()` method has higher precedence than the `+`.